### ISSUE TITLE
Clearing the UBKG cache (prod) via `/admin` results in the organ dropdown not displaying correctly

### ISSUE THREAD

**Issue by maxsibilla** [sennetconsortium/portal-ui#926]
<img width="1500" alt="image" src="https://github.com/sennetconsortium/portal-ui/assets/7218802/1bb80de8-f239-45a7-b4ec-239819872b2a">

**Comment by maxsibilla** [comment]
Request is made to https://data.sennetconsortium.org/api/ontology/C000008
Response:
```
[{
	"code": "C030024",
	"sab": "SENNET",
	"term": "Adipose Tissue"
}, {
	"code": "C030025",
	"sab": "SENNET",
	"term": "Blood "
}, {
	"code": "C030027",
	"sab": "SENNET",
	"term": "Breast"
}, {
	"code": "C030031",
	"sab": "SENNET",
	"term": "Large Intestine"
}, {
	"code": "C030032",
	"sab": "SENNET",
	"term": "Liver "
}, {
	"code": "C030052",
	"sab": "SENNET",
	"term": "Lymph Node"
}, {
	"code": "C030054",
	"sab": "SENNET",
	"term": "Pancreas"
}, {
	"code": "C030055",
	"sab": "SENNET",
	"term": "Placenta "
}, {
	"code": "C030035",
	"sab": "SENNET",
	"term": "Lung (Right)"
}, {
	"code": "C030034",
	"sab": "SENNET",
	"term": "Lung (Left)"
}, {
	"code": "C030030",
	"sab": "SENNET",
	"term": "Kidney (Right)"
}, {
	"code": "C030029",
	"sab": "SENNET",
	"term": "Kidney (Left)"
}, {
	"code": "C030041",
	"sab": "SENNET",
	"term": "Ovary (Right)"
}, {
	"code": "C030038",
	"sab": "SENNET",
	"term": "Ovary (Left)"
}, {
	"code": "C020105",
	"sab": "SENNET",
	"term": "biopsy "
}, {
	"code": "C030040",
	"sab": "SENNET",
	"term": "Skin "
}, {
	"code": "C030026",
	"sab": "SENNET",
	"term": "Brain"
}, {
	"code": "C030036",
	"sab": "SENNET",
	"term": "Muscle"
}, {
	"code": "C030039",
	"sab": "SENNET",
	"term": "Other "
}]
```

**Comment by maxsibilla** [comment]
Both `.ontology_C000008_bk` and `.ontology_C000008` look correct

**Comment by maxsibilla** [comment]
Contents of .ontology_C000008_bk

```
[{
	"code": "C030024",
	"sab": "SENNET",
	"term": "Adipose Tissue"
}, {
	"code": "C030025",
	"sab": "SENNET",
	"term": "Blood "
}, {
	"code": "C030027",
	"sab": "SENNET",
	"term": "Breast"
}, {
	"code": "C030031",
	"sab": "SENNET",
	"term": "Large Intestine"
}, {
	"code": "C030032",
	"sab": "SENNET",
	"term": "Liver "
}, {
	"code": "C030052",
	"sab": "SENNET",
	"term": "Lymph Node"
}, {
	"code": "C030054",
	"sab": "SENNET",
	"term": "Pancreas"
}, {
	"code": "C030055",
	"sab": "SENNET",
	"term": "Placenta "
}, {
	"code": "C030035",
	"sab": "SENNET",
	"term": "Lung (Right)"
}, {
	"code": "C030034",
	"sab": "SENNET",
	"term": "Lung (Left)"
}, {
	"code": "C030030",
	"sab": "SENNET",
	"term": "Kidney (Right)"
}, {
	"code": "C030029",
	"sab": "SENNET",
	"term": "Kidney (Left)"
}, {
	"code": "C030041",
	"sab": "SENNET",
	"term": "Ovary (Right)"
}, {
	"code": "C030038",
	"sab": "SENNET",
	"term": "Ovary (Left)"
}, {
	"code": "C020105",
	"sab": "SENNET",
	"term": "biopsy "
}, {
	"code": "C030040",
	"sab": "SENNET",
	"term": "Skin "
}, {
	"code": "C030026",
	"sab": "SENNET",
	"term": "Brain"
}, {
	"code": "C030036",
	"sab": "SENNET",
	"term": "Muscle"
}, {
	"code": "C030039",
	"sab": "SENNET",
	"term": "Other "
}]
```

**Comment by libpitt** [comment]
> Contents of .ontology_C000008_bk
> 
> ```
> [{
> 	"code": "C030024",
> 	"sab": "SENNET",
> 	"term": "Adipose Tissue"
> }, {
> 	"code": "C030025",
> 	"sab": "SENNET",
> 	"term": "Blood "
> }, {
> 	"code": "C030027",
> 	"sab": "SENNET",
> 	"term": "Breast"
> }, {
> 	"code": "C030031",
> 	"sab": "SENNET",
> 	"term": "Large Intestine"
> }, {
> 	"code": "C030032",
> 	"sab": "SENNET",
> 	"term": "Liver "
> }, {
> 	"code": "C030052",
> 	"sab": "SENNET",
> 	"term": "Lymph Node"
> }, {
> 	"code": "C030054",
> 	"sab": "SENNET",
> 	"term": "Pancreas"
> }, {
> 	"code": "C030055",
> 	"sab": "SENNET",
> 	"term": "Placenta "
> }, {
> 	"code": "C030035",
> 	"sab": "SENNET",
> 	"term": "Lung (Right)"
> }, {
> 	"code": "C030034",
> 	"sab": "SENNET",
> 	"term": "Lung (Left)"
> }, {
> 	"code": "C030030",
> 	"sab": "SENNET",
> 	"term": "Kidney (Right)"
> }, {
> 	"code": "C030029",
> 	"sab": "SENNET",
> 	"term": "Kidney (Left)"
> }, {
> 	"code": "C030041",
> 	"sab": "SENNET",
> 	"term": "Ovary (Right)"
> }, {
> 	"code": "C030038",
> 	"sab": "SENNET",
> 	"term": "Ovary (Left)"
> }, {
> 	"code": "C020105",
> 	"sab": "SENNET",
> 	"term": "biopsy "
> }, {
> 	"code": "C030040",
> 	"sab": "SENNET",
> 	"term": "Skin "
> }, {
> 	"code": "C030026",
> 	"sab": "SENNET",
> 	"term": "Brain"
> }, {
> 	"code": "C030036",
> 	"sab": "SENNET",
> 	"term": "Muscle"
> }, {
> 	"code": "C030039",
> 	"sab": "SENNET",
> 	"term": "Other "
> }]
> ```

This should not be the content.

**Comment by libpitt** [comment]
Fix note: `"organ_types":"C000008"` in `NEXT_PUBLIC_UBKG_CODES` must have a corresponding endpoint to retrieve ubkg result with the `rui_code` prop and others. Please add `"C000008": "organs?application_context=SENNET"` to `NEXT_PUBLIC_UBKG_CODES_PATHS`
